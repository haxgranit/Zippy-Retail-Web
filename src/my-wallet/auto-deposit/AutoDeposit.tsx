import { Button } from 'react-bootstrap';
import PageContainer from '../../common/PageContainer';

export default function AutoDeposit() {
  return (
    <PageContainer title="Auto Deposit" backdropImage="backdrop-image-3">
      <div className="body">
        <p>
          Auto Deposit is a convenient way to automatically put funds into your Zippy Wallet. Your
          employer, a government agency or other financial institutions can deposit funds directly
          into your Zippy Wallet account for you.
        </p>
        <p>
          To set up a direct deposit complete a direct deposit form and give it to those who make
          regular payments to you.
        </p>
      </div>
      <div className="action">
        <Button className="zippy-btn">Setup Direct Deposit</Button>
      </div>
    </PageContainer>
  );
}
