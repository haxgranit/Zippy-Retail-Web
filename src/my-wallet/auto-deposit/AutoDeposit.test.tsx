import AutoDeposit from './AutoDeposit';

const ReactTestRenderer = require('react-test-renderer');

describe('AutoDeposit Component', () => {
  it('matches the snapshot when loaded', () => {
    const tree = ReactTestRenderer.create(<AutoDeposit />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
