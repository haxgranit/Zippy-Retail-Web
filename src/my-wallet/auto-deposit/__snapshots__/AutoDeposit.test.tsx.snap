// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutoDeposit Component matches the snapshot when loaded 1`] = `
<div
  className=""
>
  <div
    className="zippy-cash-container-wrap"
  >
    <div
      className="backdrop backdrop-image-3"
    />
    <div
      className="zippy-cash-container"
    >
      <div
        className="zippy-cash-container-header"
      >
        <h1
          className="title"
        >
          Auto Deposit
        </h1>
        
      </div>
      <div
        className="zippy-cash-container-body"
      >
        <div
          className="body"
        >
          <p>
            Auto Deposit is a convenient way to automatically put funds into your Zippy Wallet. Your employer, a government agency or other financial institutions can deposit funds directly into your Zippy Wallet account for you.
          </p>
          <p>
            To set up a direct deposit complete a direct deposit form and give it to those who make regular payments to you.
          </p>
        </div>
        <div
          className="action"
        >
          <button
            className="zippy-btn btn btn-primary"
            disabled={false}
            type="button"
          >
            Setup Direct Deposit
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
